src/queries/incremental-compilation-in-detail.md

```diff
@@ -177,7 +177,7 @@ fn try_mark_green(tcx, current_node) -> bool {
 }
 
 // Note: The actual implementation can be found in
-//       compiler/rustc_middle/src/dep_graph/graph.rs
+//       compiler/rustc_query_system/src/dep_graph/graph.rs
```

Comment:
Shouldn't we move it outside of codeblock and add a link so that linkchecker can catch it?

Reply:
@JohnTitor https://github.com/rust-lang/rustc-dev-guide/pull/1517